packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -478,11 +479,8 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
   });
 
   describe('Initial Focus', () => {
-    before(() => {
+    beforeEach(() => {
       browser.url('/#/raw/tests/cerner-terra-framework-docs/date-picker/date-picker-default');
-      $('input[name="terra-date-month-date-input"]').setValue('12');
-      $('input[name="terra-date-day-date-input"]').setValue('18');
-      $('input[name="terra-date-year-date-input"]').setValue('1994');
```

Comment:
You need to remove all the tests in this describe block because the date picker no longer supports clearing the field with a single delete press.

Reply:
Updated - https://github.com/cerner/terra-framework/pull/2030/commits/752e707c0d3076d1cd22f145b5b362976cd3cc5e